build_run_docker.sh

```diff
@@ -658,7 +658,17 @@ fi
 echo "$BASE_DOCKER_COMPOSE_FILE" >$DOCKER_COMPOSE_FILEPATH
 echo "$CONFIG_INI" >$CONFIG_INI_FILEPATH
 
+if which docker-compose >/dev/null 2>&1; then
+    DOCKER_COMPOSE=docker-compose
+elif docker compose >/dev/null 2>&1; then
+    DOCKER_COMPOSE="docker compose"
+else
+    echo "docker compose not installed."
+    exit 1
+fi
+
+set -x
```

Comment:
Do we need this?

Reply:
Good catch, my mistake.

Reply:
I added it for my debugging and forgot to remove it before commit. Pushed a fix.